### PR TITLE
Method/fixing find user data method

### PR DESCRIPTION
**Pull request by Jedeo**
What’s this PR do?
- fixes the issue of finding user data by an array with that said, now was able to find user data by there Id's not indexed in the array

Where should the reviewer start?
-look at the `findUserData()` method in the UserRepository class
How should this be manually tested?
-you can use `npm test` I tested and did not get any problems
What are the relevant GH issues?
-there were no issues it just was finding the userData the way it was intended to.

### CODE REVIEW

File: src/UserRepository.js [hunk 1]
```diff
@@ -4,7 +4,12 @@ class UserRepository {
     this.avgUserStepGoal = null;
   }
   findUserData(id) {
-    return this.allUserData[id - 1];
+    const userData = this.allUserData.find((data) => {
+      if (data.id === id) {
```
Comment (forsethnico) [inline src/UserRepository.js]: Remove if statement, can just have data => data.id === id

File: src/scripts.js [hunk 2]
```diff
@@ -21,63 +21,32 @@ const userInfo = userRepository.findUserData(randomIndex);
...
 
 function displayFriends() {
-  const userName = user.friends.map(
-    (friend) => `<li> ${userInfo.findUserData(friend)} </li>`
-  );
-
-  friend.innerHTML += userName.reduce((acc, name) => {
-    acc = name.name;
-    return `<ul> ${acc} </ul>`;
+  user.friends.find((friend) => {
```
Comment (forsethnico) [inline src/scripts.js:46]: We don't want to use a find here since it will only find the first friend

**Review by forsethnico — CHANGES_REQUESTED** [review]
The changes in the scripts file are not needed. And the findUserData needs to be refactored to remove if statement since the return is implicitly using boolean logic

File: src/UserRepository.js (hunk 1 above)
Comment (Jedeo) [reply src/UserRepository.js]: just saw this but I have already refactored

**Comment by forsethnico** [comment]
Will close this PR and the findUserData function will be updated in a new branch once the main has been updated.

**Pull request closed without merging**